### ISSUE TITLE
Mac silicon 

### ISSUE THREAD

**Issue by timister** [pokemon-speedrunning/gambatte-speedrun#109]
Does this work on m1 mac’s with the new apple silicon chip?

**Comment by CasualPokePlayer** [comment]
r717 should just fine under Rosetta.

**Comment by GimmyTomas** [comment]
it would be nice to release a build that runs natively.

**Comment by merc1er** [comment]
> Does this work on m1 mac’s with the new apple silicon chip?

Yes I can confirm this works on Apple Silicon (tested on M1 MacBook Pro).
